plugins/c8y_log_plugin/src/main.rs

```diff
@@ -244,17 +246,20 @@ fn create_init_logs_directories_and_files(
     // creating c8y-log-plugin.toml
 
     // NOTE: file needs 775 permission or inotify can not watch for changes inside the file
-    create_file_with_user_group(
+    let status = create_file_with_user_group(
         &format!("{config_dir}/{DEFAULT_PLUGIN_CONFIG_FILE}"),
         "tedge",
         "tedge",
         0o775,
     )?;
 
-    // append default content to c8y-log-plugin.toml
-    create_default_log_plugin_file(
-        &format!("{config_dir}/{DEFAULT_PLUGIN_CONFIG_FILE}"),
-        logs_dir,
-    )?;
+    if let FileCreateStatus::CreateNew = status {
+        // append default content to c8y-log-plugin.toml
+        create_default_log_plugin_file(
+            &format!("{config_dir}/{DEFAULT_PLUGIN_CONFIG_FILE}"),
+            logs_dir,
+        )?;
+    }
```

Comment:
I'm not really happy with this API. It's okay but a bit complex for little added value: one create an empty file, then close it, and if actually new reopen it to add some content if the file were actually new. Why not a create function with a default content to be used if the file is new.